pom.xml

```diff
@@ -1185,5 +1185,39 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jdk-12+</id>
+            <activation>
+                <jdk>[12,)</jdk>
```

Comment:
I guess we can just focus on 17+ on only LTS versions

Reply:
This should be a fixed 17

Reply:
done